Makefile

```diff
@@ -7,8 +7,8 @@ CRD_OPTIONS ?= "crd"
 DOCKER_SRC=/usr/src/antrea.io/nephe
 DOCKER_GOPATH=/tmp/gopath
 DOCKER_GOCACHE=/tmp/gocache
-GENERATE_CODE_LIST={$$(go list ./... | grep -e apis/crd -e apis/runtime | paste -s -d, -)}
-GENERATE_MANIFEST_LIST={$$(go list ./... | grep apis/crd | paste -s -d, -)}
+GENERATE_CODE_LIST={$$(go list ./... | grep -e apis/crd -e apis/runtime| paste -s -d, -)}
+GENERATE_MANIFEST_LIST={$$(go list ./... | grep -E 'apis/crd|pkg/apiserver/webhook' | paste -s -d, -)}
```

Comment:
nit here too

Reply:
This is a regular expression, so space cannot be added here.